docs/docs/contribute/index.md

```diff
@@ -30,6 +35,11 @@ The command is (from the project root):
 $ make tests
 ```
 
+For [DensePose](../models/densepose.md) related tests, the command is:
+```console
+$ make densepose-tests
```

Comment:
Should this command also set `ZAMBA_RUN_DENSEPOSE_TESTS=1` before running and include those tests? Looks like this command currently just runs the cli tests:
https://github.com/drivendataorg/zamba/blob/master/Makefile#L67-L69

Reply:
good catch, fixed

Reply:
and this doesn't just run the cli test but also runs the rest of the densepose tests which are contained in `tests/test_densepose.py`